package.json

```diff
@@ -25,43 +25,46 @@
     "ci-lint": "npm run lint"
   },
   "author": "",
-  "license": "ISC",
+  "license": "Apache-2.0",
   "devDependencies": {
-    "@istanbuljs/nyc-config-typescript": "^1.0.1",
-    "@sap-cloud-sdk/generator": "1.18.1",
-    "@types/chai": "^4.2.12",
-    "@types/express": "^4.17.7",
-    "@types/mocha": "^8.0.3",
-    "@types/sinon": "^9.0.5",
-    "@types/sinon-chai": "^3.2.4",
-    "chai": "^4.2.0",
+    "@istanbuljs/nyc-config-typescript": "^1.0.2",
```

Comment:
[pp] can you perhaps add the prettier and add it ti the `lint`and `lint:fix` step. Consider doing that in a separate PR.

Reply:
Also tslint is deprecated: https://www.npmjs.com/package/tslint. Perhaps just use the same `eslint` like in the SDK package.

Reply:
Here you can also have a look on how to use our internal eslint config: https://github.tools.sap/cloudsdk/sdk-js. Remeber to disable a few rules https://github.tools.sap/cloudsdk/sdk-js/blob/main/.eslintrc.js

Reply:
I added eslint and prettier. also, I removed ts-lint. this project contains not only typescript but also javascript files. I tried to include all javascript files but can't make valid lint rules for UI5 javascript files because they have special manners. UI5 SDK provides [custom lint rules ](https://sapui5.hana.ondemand.com/sdk/#/topic/ccfe35638f1d46e2bbcf06fa60ccc6ef.html) but this doesn't work. finally I set ignore patterns for the UI5 fiels.